Failing test `test_status_all_realizations_failed` in `test_everserver.py`

```
FAILED tests/everest/test_everserver.py::test_status_all_realizations_failed@starts_everest - assert 'Optimization failed: all realizations failed.' in 'Traceback (most recent call last):\n  File "/prog/res/komodo/bleeding-20250820-1051-py311-rhel8-numpy2/root/lib64/pyt... in map_httpcore_exceptions\n    raise mapped_exc(message) from exc\nhttpx.ReadTimeout: The read operation timed out\n'
```

https://github.com/equinor/komodo-releases/actions/runs/17093481908/job/48474000729

--- a/src/ert/dark_storage/client/client.py
+++ b/src/ert/dark_storage/client/client.py
@@ -18,5 +18,8 @@ class Client(httpx.Client):
             headers = {"Token": conn_info.auth_token}
 
         super().__init__(
-            base_url=conn_info.base_url, headers=headers, verify=conn_info.cert
+            base_url=conn_info.base_url,
+            headers=headers,
+            verify=conn_info.cert,
+            timeout=15,
         )

